matrix-test.yaml

```diff
@@ -1,9 +1,11 @@
 pull-request:
   - { CUDA_VER: '11.2.2', ARCH: 'amd64', PYTHON_VER: '3.9', GPU: 'v100', DRIVER: 'earliest' }
   - { CUDA_VER: '11.8.0', ARCH: 'amd64', PYTHON_VER: '3.10', GPU: 'v100', DRIVER: 'latest' }
-  - { CUDA_VER: '11.8.0', ARCH: 'arm64', PYTHON_VER: '3.10', GPU: 'a100', DRIVER: 'latest' }
+  - { CUDA_VER: '12.0.1', ARCH: 'arm64', PYTHON_VER: '3.10', GPU: 'a100', DRIVER: 'latest' }
```

Comment:
We don't support CUDA 12 conda packages on ARM yet.

Reply:
Whoops, I forgot about this.